database/postgres/postgres.go

```diff
@@ -132,10 +140,23 @@ func (p *Postgres) Open(url string) (database.Driver, error) {
 		}
 	}
 
+	multiStatementMaxSize := DefaultMultiStatementMaxSize
```

Comment:
> The init code, and constants duplicate each other across the drivers, so probably not a bad idea to extract this into a common place (or even introduce new `MultiStatementDriver` interface).

Is this what you meant by init code?
What were you thinking in terms of a `MultiStatementDriver` interface?